lessons/Projects/project-video-game/langs/en-us/index.adoc

```diff
@@ -1,102 +1,145 @@
+| Preparation
+|
+* Following the default lesson order for Bootstrap:Algebra (e.g., teaching every lesson) will result in students building video games! That said, some teachers prefer to save all _game-focused lessons_ for the end of the year, allowing students to create games in 1-2 weeks.
+* Decide if you want to teach concepts first - and culminate with students creating their games... or take an integrated approach, where students gradually add to their games over a longer period of time. It is up to you!
+
+| Key Points for the Facilitator
+|
+* Students add to their game's code during a variety of lessons in Bootstrap:Algebra.
+* This project can be extended! Students can add their own "cheat codes" (piecewise/conditional functions). They can also add more complex movement (using quadratic, exponential, and trigonometric functions).
+
+|===
+
+== Build Your Own Video Game  @duration{flexible}
 
-=== Students will be able to...
+=== Overview
 
-* Recognize meaningful applications of multiple algebraic concepts, answering the all-important question: __"When will I ever use this?!"__
-* Express their creativity through the design of a game context
-* Take pride in having coded their first video game!
-* Share their completed video game with friends and family to play!
+This project can be used as the capstone for Bootstrap:Algebra. It is designed to provide real-world and engaging connections to the following mathematical concepts: Coordinates; Simple and Compound Inequalities; Domain and Range; Composing Functions; Connecting Representations and Defining Functions;
+Rotation, Dilation, and Translation; the Distance Formula and Pythagorean Theorem; Piecewise Functions.
 
-== Structure of the Game
 
-The code in the @starter-file{game} is divided into sections. Each section corresponds to a lesson with scaffolded activities and one or more workbook pages, which support them in writing a section of the game code. Students work in groups or pairs to customize and complete the code to make the game their own!
 
+=== Launch
 
-[cols=".^1a, .^1a, .^1a" options="header", stripes="none"]
+There are six phases of video game creation. Each phase (except for the first and the last) requires that you teach:
+
+- up to four lessons during which students learn relevant coding skills and algebra concepts
+- one lesson where students use what they've learned to update the code of their @starter-file{game, "Game Starter File"}
+
+In the first two phases (below), students conceptualize their game and then add images to their @starter-file{game, "Game Starter File"}. The table below highlights (1) the lessons you will teach to reinforce relevant algebra and coding concept, and (2) the lessons where students will update their personal video games.
+
+[cols="^.^1,<.^3,<.^6,<.^3", stripes="none", options="header"]
 |===
-| *Game Section*	| *Lessons* | *Workbook Pages*
-| *0{empty}.  Game title*
+| #
+| Phase
+| First, Learn Code and Algebra:
```

Comment:
You are very fond of the "first...then" phrasing when you write curriculum. I have to say I think it merely adds words instead of information. I'd strongly recommend retitling these as "CS and Math Concepts" and "Building the Game"

But wait! These aren't concept lists. They are _lesson lists!_ In that case the title should just be "Supporting Lessons".